src/Microsoft.AspNet.Server.Testing/Deployers/ApplicationDeployer.cs

```diff
@@ -268,5 +270,31 @@ protected static string GetOSPrefix()
                     throw new InvalidOperationException("Unrecognized operating system");
             }
         }
+
+        protected Uri FindFreeUrl(string urlHint)
```

Comment:
We may, but it's not used by anyone except implementations of ApplicationDeployer